api/src/models/student.js

```diff
@@ -0,0 +1,51 @@
+const mongoose = require("mongoose");
+const uniqueValidator = require("mongoose-unique-validator");
+/**
+ * @swagger
+ *  components:
+ *   schemas:
+ *    Student:
+ *      type: object
+ *      required:
+ *        - name
+ *        - upi
+ *      properties:
+ *        name:
+ *          type: string
+ *          description: name of the student
+ *        upi:
+ *          type: string
+ *          description: Students UPI
+ *        yearLevel:
+ *          type: integer
+ *          description: year level of the student
+ *        plans:
+ *          type: Object
+ *          description: the plans that the student has
+ */
+const studentSchema = mongoose.Schema({
+    name: {
+        type: String,
+        unique: false,
+        required: true,
+    },
```

Comment:
Should we have a student id field?

Reply:
thats what I thought but it isnt in the data model

Reply:
should i add it anyway?

Reply:
Yes - I'll add it to the data model

Reply:
ok done